libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/core/TelemetryInitializerMiddleware.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.applicationinsights.core;
+
+import com.microsoft.applicationinsights.core.dependencies.http.client.protocol.HttpClientContext;
+import com.microsoft.applicationinsights.core.dependencies.http.protocol.HttpContext;
+import com.microsoft.bot.builder.BotAssert;
+import com.microsoft.bot.builder.Middleware;
+import com.microsoft.bot.builder.NextDelegate;
+import com.microsoft.bot.builder.TelemetryLoggerMiddleware;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.schema.Activity;
+
+import java.util.concurrent.CompletableFuture;
+
+public class TelemetryInitializerMiddleware implements Middleware {
+    private HttpContext httpContext;
+    private TelemetryLoggerMiddleware telemetryLoggerMiddleware;
+    private Boolean logActivityTelemetry;
+
+    public TelemetryInitializerMiddleware(TelemetryLoggerMiddleware withTelemetryLoggerMiddleware, Boolean withLogActivityTelemetry) {
+        telemetryLoggerMiddleware = withTelemetryLoggerMiddleware;
+        logActivityTelemetry = withLogActivityTelemetry != null ? withLogActivityTelemetry : true;
+    }
+
+    public CompletableFuture<Void> onTurn(TurnContext context, NextDelegate next) {
```

Comment:
```suggestion
    public CompletableFuture<Void> onTurn(TurnContext context, NextDelegate nextTurn) {
```

Reply:
checkout that in [other](https://github.com/southworks/botbuilder-java/blob/feature/southworks/libraries/application-insights/telemetry-initializer-middleware/libraries/bot-builder/src/main/java/com/microsoft/bot/builder/TranscriptLoggerMiddleware.java#L57) middlewares it's called [next](https://github.com/southworks/botbuilder-java/blob/feature/southworks/libraries/application-insights/telemetry-initializer-middleware/libraries/bot-builder/src/main/java/com/microsoft/bot/builder/TelemetryLoggerMiddleware.java#L72)

Reply:
Ack, so let's do it as `next`, no change needed here